orchestrator/workflows/tasks/resume_workflows.py

```diff
@@ -46,14 +68,44 @@ def resume_found_workflows(waiting_process_ids: list[UUIDstr]) -> State:
             processes.resume_process(process)
             resumed_process_ids.append(process_id)
         except Exception:
-            logger.exception()
+            logger.warning()
```

Comment:
```suggestion
            logger.warning("Could not resume process", process_id=process_id, error=str(exc))
```

and change `        except Exception as exc:` above